app/components/ResultsGraph.jsx

```diff
@@ -1,11 +1,130 @@
-import React from 'react';
-import { StackedAreaChart } from 'react-native-svg-charts';
+import * as React from 'react';
+import { View } from 'react-native';
 import PropTypes from 'prop-types';
-import { Chart } from '../styles/style';
+import { StackedBarChart, YAxis, XAxis, Grid } from 'react-native-svg-charts';
+import { curveNatural } from 'd3-shape';
+import * as MathUtil from '../util/MathUtil';
 
 const ResultsGraph = ({ formData }) => {
-	console.log(formData);
-	return <StackedAreaChart style={Chart.resultsChart} />;
+	function getData() {
+		const pieData = [];
+		const date = new Date();
+		let currentYear = parseInt(date.getFullYear(), 10);
+		const maxYear =
+			parseInt(formData.mortgagePeriod.value, 10) + currentYear;
+		let balance = MathUtil.getTotalCost(formData);
+		let paid = 0;
+		const yearlyPrincipal = MathUtil.getMonthlyCost(formData) * 12;
+		while (currentYear <= maxYear) {
+			pieData.push({
+				year: currentYear,
+				balance: balance.toFixed(2),
+				paid: paid.toFixed(2)
+			});
+			balance -= yearlyPrincipal;
+
+			paid += yearlyPrincipal;
+			currentYear += 1;
+		}
+		return pieData;
+	}
+	function getYAxisData() {
+		const Data = getData();
+		const yAxisData = [];
+		Object.keys(Data).forEach(function(key) {
+			yAxisData.push(parseFloat(Data[key].balance));
+		});
+		return yAxisData;
+	}
+
+	function getXAxisData() {
+		const Data = getData();
+		const xAxisData = [];
+		Object.keys(Data).forEach(function(key) {
+			xAxisData.push(parseFloat(Data[key].year));
+		});
+		return xAxisData;
+	}
+
+	function getXAxisValue(key) {
+		const Data = getData();
+		return Data[Math.round(key)].year;
+	}
+
+	// NEED TO MAKE THIS BETTER
+	function getTicks() {
```

Comment:
This function dictates the number of labels on the X-Axis plane. Need a better implementation to determine the number of ticks.
Cannot make the number of ticks equal to the length of data within the bar chart because it results in an overflow.